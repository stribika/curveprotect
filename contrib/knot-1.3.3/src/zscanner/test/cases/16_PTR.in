$ORIGIN	.
$TTL	1

; The PTR is the same as the NS, so there are the differences and basics only.

; OK
@	PTR	test.example.com		; Relative dname
@	PTR	\# 1 00				; Hexadecimal rdata
@	TYPE12	\# 1 00				; TYPE + Hexadecimal rdata
@	TYPE12	@				; TYPE
@	ptr	@				; Type in lower-case

; KO
@	PTR
