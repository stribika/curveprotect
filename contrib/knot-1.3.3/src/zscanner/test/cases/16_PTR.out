OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000C
RDATA=0474657374076578616D706C6503636F6D00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000C
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000C
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000C
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000C
RDATA=00
------
WARNG=ZSCANNER_EBAD_RDATA
------
